build.xml

```diff
@@ -2,13 +2,26 @@
 <project name="Extra strict and opinionated rules for PHPStan" default="check">
 
 	<target name="check" depends="
+		composer-validate,
 		composer,
```

Comment:
What do you think about renaming this target to `composer-install`, for consistency?

Reply:
Yes please :)

Reply:
Done!